files/getOvfProperty.py

```diff
@@ -47,8 +47,10 @@ def main(argv):
         debug(f'ovfProperty not found: {err}')
         sys.exit(1)
 
-    # Strip enclosing quotes
-    res = re.sub(r'^(["\'])(.*)\1$', r'\2', res)
+    # Strip enclosing quotes if not a password
+    if not re.search('password', argv[0], flags=re.IGNORECASE):
```

Comment:
This is the only way I could think to detect passwords - it assumes that the OVF Property for a password will have the phrase `password` in its name.